interface/polywrap.yaml

```diff
@@ -1,4 +1,4 @@
-format: 0.3.0
+format: 0.5.0
 project:
   name: http-interface
```

Comment:
Right now we need to manually make sure the manifest's name matches the package name.